pkg/plaid_cli/linker.go

```diff
@@ -5,44 +5,75 @@ import (
 	"fmt"
 	"log"
 	"net/http"
+	"os"
 	"text/template"
 
 	"github.com/plaid/plaid-go/plaid"
 	"github.com/skratchdot/open-golang/open"
 )
 
 type Linker struct {
-	Results    chan string
-	Errors     chan error
-	Client     *plaid.Client
-	ClientOpts plaid.ClientOptions
-	Data       *Data
+	Results       chan string
+	RelinkResults chan bool
+	Errors        chan error
+	Client        *plaid.Client
+	ClientOpts    plaid.ClientOptions
+	Data          *Data
 }
 
 type TokenPair struct {
 	ItemID      string
 	AccessToken string
 }
 
-func (l *Linker) Relink(itemID string, port string) (*TokenPair, error) {
+func (l *Linker) Relink(itemID string, port string) error {
 	token := l.Data.Tokens[itemID]
-	res, err := l.Client.CreatePublicToken(token)
+	hostname, err := os.Hostname()
 	if err != nil {
-		return nil, err
+		log.Fatal(err)
 	}
-
-	return l.link(port, handleRelink(l, res.PublicToken))
+	client, err := plaid.NewClient(l.ClientOpts)
```

Comment:
Is there a reason to use `plaid.NewClient` rather than the `l.Client` object that gets passed into the constructor?

I can't quite remember why I saved `l.ClientOpts`, but I'm pretty sure it was to deal with the `"production"` vs. `"development"` environment strings below. Since those are no longer necessary and if using `l.Client` suffices, let's remove `l.ClientOpts` entirely and use `l.Client` everywhere.